src/containers/Tenant/Diagnostics/Tablets/Tablets.tsx

```diff
@@ -0,0 +1,171 @@
+import routes, {createHref} from '../../../../routes';
+import {selectTabletsWithFqdn, tabletsApi} from '../../../../store/reducers/tablets';
+import {ETabletState} from '../../../../types/api/tablet';
+import type {TTabletStateInfo} from '../../../../types/api/tablet';
+import type {TabletsApiRequestParams} from '../../../../types/store/tablets';
+import {cn} from '../../../../utils/cn';
+import {DEFAULT_TABLE_SETTINGS} from '../../../../utils/constants';
+import {calcUptime} from '../../../../utils/dataFormatters/dataFormatters';
+import {useTypedDispatch, useTypedSelector} from '../../../../utils/hooks';
+import {mapTabletStateToLabelTheme} from '../../../../utils/tablet';
+import {getDefaultNodePath} from '../../../Node/NodePages';
+
+import i18n from './i18n';
+
+const b = cn('tablets-table');
+
+const columns: DataTableColumn<TTabletStateInfo & {fqdn?: string}>[] = [
+    {
+        name: 'Type',
+        get header() {
+            return i18n('Type');
+        },
+        render: ({row}) => {
+            return (
+                <span>
+                    {row.Type} {row.Leader ? <Text color="secondary">leader</Text> : ''}
+                </span>
+            );
+        },
+    },
+    {
+        name: 'TabletId',
+        get header() {
+            return i18n('Tablet');
+        },
+        render: ({row}) => {
+            const tabletPath =
+                row.TabletId &&
+                createHref(routes.tablet, {id: row.TabletId}, {nodeId: row.NodeId, type: row.Type});
+
+            return <InternalLink to={tabletPath}>{row.TabletId}</InternalLink>;
+        },
+    },
+    {
+        name: 'State',
+        get header() {
+            return i18n('State');
+        },
+        render: ({row}) => {
+            return <Label theme={mapTabletStateToLabelTheme(row.State)}>{row.State}</Label>;
+        },
+    },
+    {
+        name: 'NodeId',
+        get header() {
+            return i18n('Node ID');
+        },
+        render: ({row}) => {
+            const nodePath = row.NodeId === undefined ? undefined : getDefaultNodePath(row.NodeId);
+            return <InternalLink to={nodePath}>{row.NodeId}</InternalLink>;
+        },
+        align: 'right',
+    },
+    {
+        name: 'FQDN',
+        get header() {
+            return i18n('Node FQDN');
+        },
+        render: ({row}) => {
+            if (!row.fqdn) {
+                return <span>—</span>;
+            }
+            return <EntityStatus name={row.fqdn} showStatus={false} hasClipboardButton />;
+        },
+    },
+    {
+        name: 'Generation',
+        get header() {
+            return i18n('Generation');
+        },
+        align: 'right',
+    },
+    {
+        name: 'Uptime',
+        get header() {
+            return i18n('Uptime');
+        },
+        render: ({row}) => {
+            return calcUptime(row.ChangeTime);
+        },
+        sortAccessor: (row) => -Number(row.ChangeTime),
```

Comment:
Please, add `align: 'right'` to `Uptime`, it makes easier row to row comparison, when values have different length (for example "16d 16:16:16" and "0:16:16")